server/lib/nl/detection/rerank.py

```diff
@@ -22,7 +22,59 @@
 
 
 def rerank(rerank_fn: RerankCallable, query: str,
-           candidates: vars.VarCandidates,
+           var_candidates: vars.VarCandidates,
            debug_logs: Dict) -> vars.VarCandidates:
-  # TODO: Implement me
-  return candidates
+  # 1. Prepare indexes and inputs
+
+  # List of query-sentence pairs.
+  qs_pairs: List[List[str, str]] = []
+  # Sentence to index into var_candidates parallel arrays.
```

Comment:
nit: mention that this is the index of the sentence's matching sv's index in var_candidates.svs

Reply:
Done